source/vistas/core/color.py

```diff
@@ -117,7 +123,7 @@ def hsva_list(self):
 
 
 def interpolate_color(range, min_color: Color, max_color: Color, t):
-    """ Linearly interpolate in the HSV color space """
+    """ Linearly interpolate in the HSV color space. Copies colors in RGB space to HSV. """
```

Comment:
If you feel that conversion implies copy is an important detail to convey, I would do that on the relevant conversion functions themselves.